build.sh

```diff
@@ -42,9 +40,15 @@ usage()
     echo "skipnative - do not build native components."
     echo "skipmscorlib - do not build mscorlib.dll."
     echo "skiptests - skip the tests in the 'tests' subdirectory."
-    echo "skiprestore - skip restoring nuget packages."
     echo "skipnuget - skip building nuget packages."
-    echo "disableoss - Disable Open Source Signing for mscorlib."
+    echo "verbose - optional argument to enable verbose build output."
+    echo "-skiprestore: skip restoring packages ^(default: packages are restored during build^)."
```

Comment:
No, there are the options that will not be directly passed to run command, which will set the necessary msbuild properties based on the information provided from config.json.. The idea is to move most of these options into run, so not much parsing has to happen in build.cmd..